src/standalone/Vegetation/canopy_parameterizations.jl

```diff
@@ -263,7 +263,7 @@ function canopy_sw_rt_two_stream(
     h = √(b^2 - c^2) / μ̄
     σ = (μ̄ * K)^2 + c^2 - b^2
 
-    u₁ = b - c / α_soil
+    u₁ = b - c / max(α_soil, eps(FT))
```

Comment:
oh, we dont need this unless alpha_soil = 0! so it should be OK in this limit.

Reply:
We can just add at the very top of the function:
\alpha_soil = max(eps(FT), alpha_soil)